tests/functional/configure/test_addmodel.py

```diff
@@ -82,3 +82,12 @@ def test_add_model_with_service_name(self):
             os.path.exists(os.path.join(
                 self.customer_data_root, 'override-name', '2015-12-02',
                 'service-2.json')))
+
+
+class TestConfigureSSO(BaseAWSCommandParamsTest):
```

Comment:
Let's add this test in a new `test_sso.py` file in the `configure` functional test suite instead of using the `test_addmodel.py`. The tests are typically organized by which command they apply to so it is weird to be adding a test case for `configure sso` to a test file meant only for `configure add-model`